services/spar/src/Spar/Sem/BrigAccess/Http.hs

```diff
@@ -26,6 +26,7 @@ import Polysemy
 import Polysemy.Error (Error)
 import Spar.Error (SparError)
 import qualified Spar.Intra.Brig as Intra
+import qualified Spar.Intra.Brig as Itra
```

Comment:
```suggestion
```

hehe, stupid ide...  :-)

Reply:
Indeed: I made the `Itra` typo and the IDE fixed it... :rofl: